tests/testcases/utils.py

```diff
@@ -0,0 +1,5 @@
+import django
+
+
+def is_django_gte_19():
+    return django.VERSION >= (1, 9)
```

Comment:
Also, this should be in the [`compat`](https://github.com/incuna/incuna-test-utils/blob/master/incuna_test_utils/compat.py) module.